core_data_modules/cleaners/location_tools/kenya_locations.py

```diff
@@ -25,26 +25,56 @@ def is_location_code(cls, text):
         :return: Whether or not location matches a known kenya location code
         :rtype: bool
         """
-        return text in KenyaCodes.CONSTITUENCIES or \
+        return text in KenyaCodes.WARDS or text in KenyaCodes.CONSTITUENCIES or \
                text in KenyaCodes.COUNTIES
 
-    @staticmethod
-    def constituency_for_location_code(location_code):
+    @classmethod
+    def ward_for_location_code(cls, location_code):
         """
-        Returns the Kenya constituency for the provided Kenya location code.
+        Returns the Kenya ward for the provided Kenya location code.
 
         The provided code may be a Kenya constituency only.
 
-        >>> KenyaLocations.constituency_for_location_code(KenyaCodes.NORTH_IMENTI)
-        'north imenti'
-        >>> KenyaLocations.constituency_for_location_code(KenyaCodes.NAIROBI)
+        >>> KenyaLocations.ward_for_location_code(KenyaCodes.KIVAA)
+        'kivaa'
+        >>> KenyaLocations.ward_for_location_code(KenyaCodes.NAIROBI)
         'NC'
 
-        :param location_code: A Kenya constituency code
+        :param location_code: A Kenya ward code
+        :type location_code: str
+        :return: Kenya ward or Codes.NOT_CODED
+        :rtype: str
+        """
+        if location_code in KenyaCodes.WARDS:
+            return location_code
+
+        return Codes.NOT_CODED
+
+
+    @classmethod
+    def constituency_for_location_code(cls, location_code):
+        """
+        Returns the constituency for the provided Kenya location code.
+
+        The provided code may be a Kenya ward or a constituency.
+
+        >>> KenyaLocations.constituency_for_location_code(KenyaCodes.KIVAA)
+        'masinga'
+        >>> KenyaLocations.constituency_for_location_code(KenyaCodes.MASINGA)
+        'masinga'
+
+        :param location_code: A Kenya ward or constituency code
         :type location_code: str
-        :return: Kenya constituency or Codes.NOT_CODED
+        :return: A kenyan constituency code or Codes.NOT_CODED
```

Comment:
```suggestion
        :return: Kenya constituency or Codes.NOT_CODED
```

Reply:
For consistency with other functions i.e., `ward_for_location_code`, `Somalia district or Codes.NOT_CODED`